### PR TITLE
allow (\n{ in brace_linter

### PR DESCRIPTION
**Pull request by AshesITR**
quick fix to #487 (also allow `(\n{`)

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
 * Rename `semicolon_terminator_linter` to `semicolon_linter` for better consistency. `semicolon_terminator_linter` survives but is marked for deprecation. The new linter also has a new signature, taking arguments `allow_compound` and `allow_trailing` to replace the old single argument `semicolon=`, again for signature consistency with other linters.
 * Combined several curly brace related linters into a new `brace_linter` (#1041, @AshesITR):
   + `closed_curly_linter()`, also allowing `}]` in addition to `})` and `},` as exceptions.
-  + `open_curly_linter()`, no longer linting unnecessary trailing whitespace and also allowing `,` and `%>%` on
+  + `open_curly_linter()`, no longer linting unnecessary trailing whitespace and also allowing `(`, `,` and `%>%` on
```
Comment (MichaelChirico) [inline NEWS.md:14]: `open_curly_linter` doesn't throw a lint on your test case below on the CRAN version -- IINM this was a regression introduced by the move to XPath in `brace_linter`. Can you confirm that and if so, we don't need a NEWS item right?

Comment (AshesITR) [reply NEWS.md:14]: Can you verify using the `open_curly_linter()` implementation? It's deprecated but still there and the old code should lint due to `!some_before`...
Or was there another change to open_curly_linter in the mean time?

Comment (MichaelChirico) [reply NEWS.md:14]: I still don't have a computer set up to test. maybe this weekend.

Comment (AshesITR) [reply NEWS.md:14]:
```
> packageVersion("lintr")
[1] ‘2.0.1.9000’
> lintr::lint("tryCatch(\n  {\n    print(1)\n  },\n  error = function(err) {\n  }\n)", linters = list(lintr::open_curly_linter()))
Warning: Linter open_curly_linter was deprecated in lintr version 2.0.1.9001. Use brace_linter instead.
<text>:2:3: style: Opening curly braces should never go on their own line and should always be followed by a new line.
  {
  ^
```

Comment (AshesITR) [reply NEWS.md:14]: I just noticed I tested on the dev version (oops).
CRAN has the same behaviour, though:

```
> packageVersion("lintr")
[1] ‘2.0.1’
> lintr::lint("tryCatch(\n  {\n    print(1)\n  },\n  error = function(err) {\n  }\n)", linters = list(lintr::open_curly_linter()))
<text>:2:3: style: Opening curly braces should never go on their own line and should always be followed by a new line.
  {
  ^
```

Comment (MichaelChirico) [reply NEWS.md:14]: thanks for confirming

**Review by MichaelChirico — APPROVED** [review]

**Pull request merged**